models/ntp

```diff
@@ -0,0 +1,61 @@
+---
+
+GENERATOR_VERSION: '1.0'
+
+ANSIBLE_METADATA: |
+  {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'network'
+  }
+
+NETWORK_OS: vyos
+RESOURCE: ntp
+COPYRIGHT: Copyright 2021 Red Hat
+
+
+DOCUMENTATION: |
+  module: vyos_ntp
+  version added:
+  short_description:  Manages ntp modules of Vyos network devices
+  description:  This module manages ntp configuration on devices running Vyos network devices
+  author: Varshitha Yataluru (@vyatalur)
+  options:
+    config:
+        description: List of configurations for ntp module
+        type: list
+        elements: dict
+        suboptions:
+            allow-clients:
+                description: Network Time Protocol (NTP) server options
+                type: str
```

Comment:
this file is in the wrong place in the repo dir you need to remove it from the PR